index.js

```diff
@@ -1,25 +1,51 @@
 'use strict';
 
+const modules = [require('babel-plugin-transform-es2015-modules-commonjs'), {
+  strict: false,
+}];
+
+const defaultTargets = {
+  android: 30,
+  chrome: 35,
+  edge: 14,
+  explorer: 9,
+  firefox: 52,
+  safari: 8,
+  ucandroid: 1,
+};
+
+function buildTargets(options) {
+  return Object.assign({}, defaultTargets, options.additionalTargets);
+}
+
 module.exports = function buildAirbnbPreset(context, options) {
-  var preset; // eslint-disable-line no-var
-  if (options && options.modules === false) {
-    preset = require('babel-preset-es2015').buildPreset(null, { modules: false });
-  } else {
-    preset = require('babel-preset-es2015-without-strict');
-  }
+  const targets = (options && options.targets) || buildTargets(options);
```

Comment:
@ljharb heads up, lint complained about the use of && and || so I wrapped it in parens.